src/components/utils/_vars.scss

```diff
@@ -5,6 +5,7 @@ $blue: blue;
 $mainFont: 'Playfair Display', serif;
 
 //screens
+$mobil: 'max-width: 475px';
```

Comment:
It has to be different. Both - with naming and with a value.
First of all `mobile` not `mobil`. And usually mobile is something by default. Yes, we might need to have different shapes of phones at times, thin we can try a different naming. Like `mediumPhone`, `largePhone` or `xPhone`, `xlPhone`.
Secondly, we discussed this only this morning, you have to avoid using `max-width`. We can proceed with this bullshit for now, but for the next project you'll get a second yellow card for this 🟨

Reply:
I try to find a positive thing in your message. And now I see that we are talking about our next project, and that means I steel have a chance to become a programmer :)